unicode.go

```diff
@@ -2,4 +2,6 @@
}-\\x{C83}\\x{C85}-\\x{DF3}\\x{DF5}-\\x{E4E}\\x{E50}-\\x{E59}\\x{E5C}-\\x{F03}\\x{F13}\\x{F15}-\\x{F84}\\x{F86}-\\x{FCF}\\x{FD5}-\\x{FD8}\\x{FDB}-\\x{1049}\\x{1050}-\\x{10FA}\\x{10FC}-\\x{135F}\\x{1369}-\\x{166D}\\x{166F}-\\x{167F}\\x{1681}-\\x{16EA}\\x{16EE}-\\x{1734}\\x{1737}-\\x{17D3}\\x{17D7}\\x{17DB}-\\x{17FF}\\x{1806}\\x{180B}-\\x{1943}\\x{1946}-\\x{1A1D}\\x{1A20}-\\x{1A9F}\\x{1AA7}\\x{1AAE}-\\x{1B59}\\x{1B61}-\\x{1BFB}\\x{1C00}-\\x{1C3A}\\x{1C40}-\\x{1C7D}\\x{1C80}-\\x{1CBF}\\x{1CC8}-\\x{1CD2}\\x{1CD4}-\\x{1FFF}\\x{200B}-\\x{2015}\\x{2018}-\\x{201F}\\x{202A}-\\x{202E}\\x{2039}-\\x{203A}\\x{203F}-\\x{2040}\\x{2044}-\\x{2046}\\x{2052}\\x{2054}\\x{2060}-\\x{2CF8}\\x{2CFD}\\x{2D00}-\\x{2D6F}\\x{2D71}-\\x{2DFF}\\x{2E02}-\\x{2E05}\\x{2E09}-\\x{2E0A}\\x{2E0C}-\\x{2E0D}\\x{2E17}\\x{2E1A}\\x{2E1C}-\\x{2E1D}\\x{2E20}-\\x{2E29}\\x{2E2F}\\x{2E3A}-\\x{2E3B}\\x{2E40}\\x{2E42}\\x{2E50}-\\x{2E51}\\x{2E53}-\\x{2FFF}\\x{3004}-\\x{303C}\\x{303E}-\\x{30FA}\\x{30FC}-\\x{A4FD}\\x{A500}-\\x{A60C}\\x{A610}-\\x{A672}\\x{A674}-\\x{A67D}\\x{A67F}-\\x{A6F1}\\x{A6F8}-\\x{A873}\\x{A878}-\\x{A8CD}\\x{A8D0}-\\x{A8F7}\\x{A8FB}\\x{A8FD}-\\x{A92D}\\x{A930}-\\x{A95E}\\x{A960}-\\x{A9C0}\\x{A9CE}-\\x{A9DD}\\x{A9E0}-\\x{AA5B}\\x{AA60}-\\x{AADD}\\x{AAE0}-\\x{AAEF}\\x{AAF2}-\\x{ABEA}\\x{ABEC}-\\x{D7FF}\\x{F900}-\\x{FDCF}\\x{FDF0}-\\x{FE0F}\\x{FE17}-\\x{FE18}\\x{FE1A}-\\x{FE2F}\\x{FE31}-\\x{FE44}\\x{FE47}-\\x{FE48}\\x{FE4D}-\\x{FE4F}\\x{FE53}\\x{FE58}-\\x{FE5E}\\x{FE62}-\\x{FE67}\\x{FE69}\\x{FE6C}-\\x{FF00}\\x{FF04}\\x{FF08}-\\x{FF09}\\x{FF0B}\\x{FF0D}\\x{FF10}-\\x{FF19}\\x{FF1C}-\\x{FF1E}\\x{FF21}-\\x{FF3B}\\x{FF3D}-\\x{FF60}\\x{FF62}-\\x{FF63}\\x{FF66}-\\x{FFEF}\\x{10000}-\\x{100FF}\\x{10103}-\\x{1039E}\\x{103A0}-\\x{103CF}\\x{103D1}-\\x{1056E}\\x{10570}-\\x{10856}\\x{10858}-\\x{1091E}\\x{10920}-\\x{1093E}\\x{10940}-\\x{10A4F}\\x{10A59}-\\x{10A7E}\\x{10A80}-\\x{10AEF}\\x{10AF7}-\\x{10B38}\\x{10B40}-\\x{10B98}\\x{10B9D}-\\x{10F54}\\x{10F5A}-\\x{11046}\\x{1104E}-\\x{110BA}\\x{110BD}\\x{110C2}-\\x{1113F}\\x{11144}-\\x{11173}\\x{11176}-\\x{111C4}\\x{111C9}-\\x{111CC}\\x{111CE}-\\x{111DA}\\x{111DC}\\x{111E0}-\\x{11237}\\x{1123E}-\\x{112A8}\\x{112AA}-\\x{1144A}\\x{11450}-\\x{11459}\\x{1145C}\\x{1145E}-\\x{114C5}\\x{114C7}-\\x{115C0}\\x{115D8}-\\x{11640}\\x{11644}-\\x{1165F}\\x{1166D}-\\x{1173B}\\x{1173F}-\\x{1183A}\\x{1183C}-\\x{11943}\\x{11947}-\\x{119E1}\\x{119E3}-\\x{11A3E}\\x{11A47}-\\x{11A99}\\x{11A9D}\\x{11AA3}-\\x{11C40}\\x{11C46}-\\x{11C6F}\\x{11C72}-\\x{11EF6}\\x{11EF9}-\\x{11FFE}\\x{12000}-\\x{1246F}\\x{12475}-\\x{16A6D}\\x{16A70}-\\x{16AF4}\\x{16AF6}-\\x{16B36}\\x{16B3C}-\\x{16B43}\\x{16B45}-\\x{16E96}\\x{16E9B}-\\x{16FE1}\\x{16FE3}-\\x{1BC9E}\\x{1BCA0}-\\x{1DA86}\\x{1DA8C}-\\x{1E95D}\\x{1E960}-\\x{1FFFD}\\x{20000}-\\x{2FFFD}\\x{30000}-\\x{3FFFD}\\x{40000}-\\x{4FFFD}\\x{50000}-\\x{5FFFD}\\x{60000}-\\x{6FFFD}\\x{70000}-\\x{7FFFD}\\x{80000}-\\x{8FFFD}\\x{90000}-\\x{9FFFD}\\x{A0000}-\\x{AFFFD}\\x{B0000}-\\x{BFFFD}\\x{C0000}-\\x{CFFFD}\\x{D0000}-\\x{DFFFD}\\x{E1000}-\\x{EFFFD}"
```

Comment:
in this case I do think that avoiding escape sequences where possible would be nice :)

Reply:
I think you can use https://pkg.go.dev/unicode#IsGraphic to determine whether a rune can be used in a string literal without escaping.